presto-native-execution/src/test/java/com/facebook/presto/nativeworker/HiveExternalWorkerQueryRunner.java

```diff
@@ -15,17 +15,24 @@
 
 import com.facebook.airlift.log.Logger;
 import com.facebook.airlift.log.Logging;
+import com.facebook.presto.testing.QueryRunner;
 import com.facebook.presto.tests.DistributedQueryRunner;
 
-public class HiveExternalWorkerQueryRunner
-{
-    private HiveExternalWorkerQueryRunner() {}
+public class HiveExternalWorkerQueryRunner {
+    private HiveExternalWorkerQueryRunner() {
+    }
+
     public static void main(String[] args)
-            throws Exception
-    {
-        // You need to add "--user user" to your CLI for your queries to work
+            throws Exception {
+        // You need to add "--user user" to your CLI for your queries to work.
         Logging.initialize();
 
+        // Create tables before launching distributed runner.
+        QueryRunner javaQueryRunner = PrestoNativeQueryRunnerUtils.createJavaQueryRunner();
+        NativeQueryRunnerUtils.createAllTables(javaQueryRunner);
+        javaQueryRunner.close();
+
+        // Launch distributed runner.
         DistributedQueryRunner queryRunner = (DistributedQueryRunner) PrestoNativeQueryRunnerUtils.createQueryRunner();
```

Comment:
This formatting looks strange too.

Reply:
The one liner? Why strange?

Reply:
It looks not like 1-liner is the silly GH UI. :)